src/plugins/intel_npu/src/compiler_adapter/src/graph.cpp

```diff
@@ -47,10 +47,13 @@ size_t Graph::export_blob(std::ostream& stream) const {
         _zeGraphExt->getGraphBinary(_handle, blob, blobPtr, blobSize);
     } else {  // in all other cases, the blob is handled by the plugin
         blobPtr = static_cast<const uint8_t*>(_blob->data());
-        blobSize = _blob->get_byte_size();
+        blobSize = this->_blob->get_byte_size();
```

Comment:
```suggestion
        blobSize = _blob->get_byte_size();
```

Is this really needed?

Reply:
![image](https://github.com/user-attachments/assets/4d54a89d-a421-43b1-996c-253b3432cbbe)